packages/validation-html/src/validate-binding-behavior.ts

```diff
@@ -60,7 +60,13 @@ export enum ValidationTrigger {
   changeOrFocusout = 'changeOrFocusout',
 }
 
+export enum CacheBindingPropertyInfo {
```

Comment:
This should be a boolean. I cannot imagine supporting more behaviour, given the context.